sda-commons-server-auth/src/main/java/org/sdase/commons/server/auth/AuthBundle.java

```diff
@@ -130,6 +139,10 @@ private KeySource createKeySources(KeyLocation keyLocation, Client client) {
     }
   }
 
+  private void setupManagedThreadManager(Environment environment) {
+    environment.lifecycle().manage(onShutdown(() -> keyLoaderScheduler.shutDown()));
```

Comment:
As you get the executor service from the lifecycle, it is already managed and will be shut down by the framework.
Although it doesn't harm the shutdown, this call of `shutDown()` is not necessary and should be removed.

Reply:
Done! My look on the `ManagedShutdownListener` definitely was too superficial. Now I think I got how it works (and removed it from the AuthBundle).